README.md

```diff
@@ -37,8 +37,7 @@ All the development tools are provided when developing inside the dev container
 also include a Python virtual environment where all the Python packages needed
 are already installed.
 
-If you decide the develop outside of the dev container, some of the development
-tools can be installed by running:
+To setup these tools and install the project dependencies, run:
```

Comment:
This step needs to be run even when developing inside the devcontainer.

Reply:
Actually this script is run when starting the container. I will revert the change and clarify.